website/docs/r/data_factory_linked_service_azure_blob_storage.html.markdown

```diff
@@ -114,9 +114,13 @@ The following supported arguments are common across all Azure Data Factory Linke
 
 The following supported arguments are specific to Azure Blob Storage Linked Service:
 
-* `connection_string` - (Optional) The connection string. Conflicts with `sas_uri` and `service_endpoint`.
+* `connection_string` - (Optional) The connection string. Conflicts with `insecure_connection_string`, `sas_uri` and `service_endpoint`.
 
-* `sas_uri` - (Optional) The SAS URI. Conflicts with `connection_string` and `service_endpoint`.
+* `insecure_connection_string` - (Optional) The connection string sent insecurely. Conflicts with `connection_string`, `sas_uri` and `service_endpoint`.
+
+~> **Note:** `connection_string` is sent to the Azure encrypted whilst `insecure_connection_string` is sent unencrypted. The benefit of using `insecure_connection_string` is that the storage account name is now viewable in state and portal.
```

Comment:
i think we should be more explicit here:
```suggestion
~> **Note:** `connection_string` uses the Azure [SecureString](https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.management.datafactory.models.securestring) to encrypt the contents within the REST payload to azure whilst the  `insecure_connection_string` is sent as a regular string. Both properties are still sent using SSL/HTTPS. At this time the portal will not decrypt Secure Strings so the `connection_string` property will show as `******` whilst `insecure_connection_string` will be viewable in the portal.
```

Reply:
Agreed